_inc/client/components/jetpack-notices/style.scss

```diff
@@ -4,4 +4,10 @@
 			font-size: rem( 12px );
 		}
 	}
+}
+
+.jp-unlinked-notice {
+	.gridicon {
+		margin-bottom: -7px;
+	}
```

Comment:
There should be some common rule, because otherwise we're having two rules: this one and  https://github.com/Automattic/jetpack/pull/7606/files#diff-6d677eefb190cb5c13ec35af47006b67